output/__pyfunceble_origin__/.gitignore

```diff
@@ -1,7 +0,0 @@
-*
```

Comment:
Please reverse this change. It is needed for development. This directory `__pyfunceble_origin__` is automatically read for the generation of `PyFunceble/data/infrastructure/dir_structure_production.json` while running the `production-pyfunceble` tool.